plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -206,7 +207,37 @@ public static void rootConfigGeneration() throws Exception {
             final Mapping mapping = config.asMapping();
             final List<Map.Entry<String, CNode>> entries = new ArrayList<>(mapping.entrySet());
             for (Map.Entry<String, CNode> entry : entries) {
-                System.out.println(entry.getKey() + " " + entry.getValue().toString());
+                System.out.println(entry.getKey());
+            }
+            System.out.println("End of an iteration of configurators");
+        }
+    }
+
+    public static void storeConfiguratorNames() {
```

Comment:
@timja I am not sure this is necessary anymore

Reply:
remove if not needed